pkg/linter/linterExecute.go

```diff
@@ -84,14 +94,20 @@ func LinterExecute(configs map[string]*vpcmodel.VPCConfig, printAllFindings bool
 		if !enable {
 			continue
 		}
-		thisLintStr := ""
 		err := thisLinter.check()
 		if err != nil {
-			return false, "", err
+			return nil, err
 		}
+	}
+	return linters, nil
+}
+
+func (linters Linters) String(printAllFindings bool) (resString string) {
+	strPerLint := []string{}
+	for _, thisLinter := range linters {
+		thisLintStr := ""
```

Comment:
```suggestion
```
Redundant line

Reply:
right. removed (and replace = in  := where needed)